doc/classes/StreamPeerTCP.xml

```diff
@@ -29,6 +29,15 @@
 				Disconnect from host.
 			</description>
 		</method>
+		<method name="set_no_delay">
+			<return type="void">
+			</return>
+			<argument index="0" name="enabled" type="bool">
+			</argument>
+			<description>
+				Disable Naggle Algorithm.
```

Comment:
I feel like some more details could be useful. I would need to google "Naggle algorithm" to understand what this method does, as I'm not a TCP expert. It's also not clear to untrained eyes how "no deplay" and "Naggle algorithm" are linked.

Reply:
Now it is better described and with a warning that disabling this, you will spend more bandwidth.